addons/crm_iap_lead_website/models/crm_reveal_rule.py

```diff
@@ -113,13 +113,13 @@ def _compute_lead_count(self):
     def action_get_lead_tree_view(self):
         action = self.env["ir.actions.actions"]._for_xml_id("crm.crm_lead_all_leads")
         action['domain'] = [('id', 'in', self.lead_ids.ids), ('type', '=', 'lead')]
-        action['context'] = dict(self._context, create=False)
+        action['context'] = dict(action.get('context', {}), create=False)
```

Comment:
This one is a real bugfix, since the action has a context, see https://github.com/odoo-dev/odoo/blob/60da4eeec324f2097c52f00793222c067aef756b/addons/crm/views/crm_lead_views.xml#L875-L887